core/src/main/resources/reference.conf

```diff
@@ -57,6 +57,8 @@ submissionmonitor {
   trackDetailedSubmissionMetrics = true
   attributeUpdatesPerWorkflow = 20000
   enableEmailNotifications = true
+  # how many workflows should the submission monitor attempt to process and update at once?
+  workflowsPerMonitorPass = 20
```

Comment:
what is a good initial value for this setting?

Reply:
Less than 600!